src/metabase/query_processor.clj

```diff
@@ -9,6 +9,7 @@
             (metabase [config :as config]
                       [db :as db]
                       [driver :as driver])
+            [metabase.models.database :as database]
             (metabase.models [field :refer [Field]]
                              [query-execution :refer [QueryExecution]])
```

Comment:
yeah, i know you like that approach, but after working in Clojure for a while i find that i don't care very much for that syntax.  it's easier in my IDE if all of the requires are individual rather than nested and so i prefer that approach.